components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -120,18 +117,8 @@ public Application getApplication(String clientId) throws DCRMException {
         Application application = buildResponse(consumerAppDTO, tenantDomain);
         application.setExtAllowedAudience(serviceProvider.getAssociatedRolesConfig().getAllowedAudience());
 
-        String attributeFilterName = IdentityUtil.getProperty(OAuthConstants.ADDITIONAL_ATTRIBUTE_FILTER);
```

Comment:
why are we removing the identity.xml file reading? Isn't there any case where customers have the customization which is not an OSGI service?

Reply:
Even though the config is there, it is not working in the latest update levels as well. Hence, it won't be working for customers as well.

Reply:
Was this working in the default 7.0.0 pack? Don't we need to do the fix in a way that should work with non-osgi and osgi?

Reply:
This was added as a patch for IS 7.0. And was not working when the extension was from another bundle. Hence I dont think there is an issue

Reply:
Ok does that meant this has never worked before? Even when it was added initially?

Reply:
Yes thats correct. But if the extension is in the same module it works.